lib/lightning_web/live/project_live/settings.ex

```diff
@@ -665,7 +686,7 @@ defmodule LightningWeb.ProjectLive.Settings do
       <div class="px-6">
         <p class="text-sm text-gray-500">
           Are you sure you want to remove "{@project_user.user.first_name} {@project_user.user.last_name}" from this project?
-          They will nolonger have access.
+          They will no longer have access{@credentials_text} will be removed from this project.
```

Comment:
Hmm 🤔 , this will read: `They will no longer have access will be removed from this project` when the user has no credential in the project.  Could you double check?

Reply:
Thanks for the catch, this is fixed